package.json

```diff
@@ -21,10 +21,11 @@
     "build-desktop": "cross-env REACT_APP_BUILD_FOR_DESKTOP=true REACT_APP_ENV=production yarn build-mainnet",
     "build-mainnet": "cross-env REACT_APP_NETWORK=mainnet yarn build",
     "build": "cross-env REACT_APP_APP_VERSION=$npm_package_version rescripts --max-old-space-size=8192 build",
+    "compile-electron": "rm -rf .public/dist && tsc --project tsconfig.electron.json",
     "copy-gateway-types": "rm -rf src/types/gateway; cp -r node_modules/@gnosis.pm/safe-react-gateway-sdk/dist/types src/types/gateway",
     "eject": "rescripts eject",
-    "electron-build": "electron-builder --mac --windows --linux",
-    "electron-dev": "concurrently \"BROWSER=none yarn start\" \"wait-on http://localhost:3000 && electron .\"",
+    "electron-build": "yarn compile-electron && electron-builder --mac",
```

Comment:
Just mac now?

Reply:
The command above wasn't working any way. You can only trigger a mac build on a mac, a windows build on a windows and a linux build on a linux.

For me to have as reference is enough and we don't need to have the three commands there, we don't use them that much